<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QTabWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="tabsClosable">
   <bool>true</bool>
  </property>
  <property name="movable">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="globalChat">
   <attribute name="title">
    <string>Global Chat</string>
   </attribute>
   <widget class="QLineEdit" name="globalChatLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="globalChatPushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="globalChatTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>311</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="localChat">
   <attribute name="title">
    <string>Local Chat</string>
   </attribute>
   <widget class="QLineEdit" name="localChatLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="localChatPushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="localChatTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>311</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
